app/controllers/documents_controller.rb

```diff
@@ -22,6 +26,17 @@ def create
     end
   end
 
+  def update
+    @document = authorize Document.find(params[:id])
+    if update_transaction.success?
+      flash[:notice] = t 'activerecord.notices.messages.update_success'
```

Comment:
- Try the lookup `t '.success'`. You will then have an idea on where to put that key. The `activerecord` namespace isn't appropriate here, as we are not in an ActiveRecord object.
- On how to organize locale files, the easiest to maintain is to mimic the actual project structure. So that lookup key would ideally go in a file called `config/locales/controllers/documents.fr.yml`. Like that, we know exactly where are the locale file for a given object.